app/src/main/java/app/nexd/android/ui/helper/requestOverview/HelperOverviewFragment.kt

```diff
@@ -52,31 +55,47 @@ class HelperOverviewFragment : Fragment() {
             HelpRequestBinder()
         )
         nearRequestsAdapter.registerItemBinders(
-            AvailableRequestBinder()
+            HelpRequestBinder()
         )
 
         viewModel.run {
             getMyAcceptedRequests().observe(viewLifecycleOwner, Observer { myAcceptedRequests ->
 
                 updateAcceptedRequests(myAcceptedRequests)
 
-                val title = context?.getString(R.string.helper_request_overview_heading_accepted_section) ?: ""
-                val small = "(${myAcceptedRequests.size} / 20)"
-                val acceptedTitle = SpannableString(title + small)
-                acceptedTitle.setSpan(
-                    RelativeSizeSpan(0.5f), title.length,
-                    title.length + small.length, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE
+                val acceptedTitle =
+                    resources.getString(R.string.helper_request_overview_heading_accepted_section)
+                val acceptedInfo = " (${myAcceptedRequests.size} / 20)"
```

Comment:
we should start using format strings for that. also we should talk about the limit of 20. at the moment we display this limit but don't validate it on request accept. I am not sure if we need this limit at all